i18n/en-US.yml

```diff
@@ -36,10 +36,40 @@ components:
     tripDurationFormatZeroHours: "{minutes, number} min"
     # TODO: Distinguish between one hour (singular) and 2 hours or more?
     tripDurationFormat: "{hours, number} hr {minutes, number} min"
+  RouteDetails:
+    operatedBy: "Operated by {agencyName}"
+    moreDetails: "More Details"
+    stopsTo: "Towards"
+    selectADirection: "Select a direction..."
+  RouteViewer:
+    allAgencies: All Agencies
+    allModes: All Modes # Note to translator: This text is width-constrained.
+    findARoute: Find A Route
+    noFilteredRoutesFound: No routes match your filter!
+    noRouteUrl: No route URL provided.
+    title: Route Viewer
+    shortTitle: View Routes
+    agencyFilter: Agency Filter
+    modeFilter: Mode Filter
+    details: " " # If the string is left blank, React-Intl renders the id
+  RouteRow:
+    operatorLogoAltText: '{operatorName} logo'
+  TransitVehicleOverlay:
+    # keys designed to match API output
+    incoming_at: "approaching {stop}"
+    stopped_at: "doors open at {stop}"
+    in_transit_to: "next stop {stop}"
+
+    vehicleName: "Vehicle {vehicleNumber}: "
+    realtimeVehicleInfo: "<b>{vehicleNameOrBlank}</b>{relativeTime}"
+    travelingAt: "traveling at {milesPerHour}"
   AppMenu:
-    closeMenu: "Close Menu"
-    openMenu: "Open Menu"
-    menuItemIconAlt: "Icon for {label} menu item"
+     closeMenu: "Close Menu"
+     openMenu: "Open Menu"
+     menuItemIconAlt: "Icon for {label} menu item"
+     callHistory: "Call History"
+     fieldTrip: "Field Trip"
+     mailables: "Mailables"
```

Comment:
Extra indent not needed

Reply:
Fixed in da3a686bd0dcfa48cfbdfb3b1052864eab9933fa